### ISSUE TITLE
MMap crash on Windows7 x32

### ISSUE THREAD

**Issue by SLIpros** [DarthTon/Blackbone#231]
Hey, it gives me such log:
```
Failed to load msdia140.dll, error 0x087e
LdrpHandleTlsData64 not found
LdrKernel32PatchAddress not found
APC64PatchAdress not found
```
![screenshot2](https://user-images.githubusercontent.com/6332783/35103026-c3e9f3da-fc75-11e7-9bce-905745c8aa9e.png)

and then crash exe.
All working fine for me (Windows 10 Pro), but my friend that is on Windows 7 x32 has such error.
Any ideas what is that? Thank you for any reply.

**Comment by DarthTon** [comment]
x86 OS aren't officially supported. I can check what's wrong when I have time.

**Comment by SLIpros** [comment]
All was fine, but when i update to last BB version i got this issue. Will try to downgrade and report here if that change something.

**Comment by mrgreywater** [comment]
Well, the first problem seems to stem from the fact that the distributed msdia140.dll is compiled for x64. I wonder wouldn't it possible to download the pdb's via WinINet and read the symbols from the pdb with the dbghelp.dll instead, which is distributed with all versions of windows already?

**Comment by SLIpros** [comment]
@mrgreywater i tryed to put msdia140.dll near exe, than error line (Failed to load msdia140.dll, error 0x087e) disappear, but exe still crashing.

**Comment by DarthTon** [comment]
PDB shuldn't be an issue. I have both x86 and x64 msdia140.dll in repo. Besides if it fails to load msdia, blackbone falls back to pattern scan.
@mrgreywater it's possible to do a direct .pdb download, but I'm lazy and msdia handles GUID stuff for me.

**Comment by DarthTon** [comment]
Possibly fixed in 80b8cd2d1ba1dbbee41058a8e8195fc2bdac9945
